recipe/meta.yaml

```diff
@@ -14,28 +14,28 @@ source:
     - cmake.patch  # [win]
 
 build:
-  number: 1
-  skip: true  # [win and vc<14]
+  number: 0
+  skip: True  # [win and vc<14]
   run_exports:
     - {{ pin_subpackage(name, max_pin='x.x') }}
-
-# TODO: conda-forge giflib missing headers on windows?
+  ignore_run_exports:
+    - xz
 
 requirements:
   build:
     - {{ compiler('c') }}
-    - autoconf  # [not win]
-    - automake  # [not win]
-    - make  # [not win]
-    - libtool  # [not win]
-    - m4  # [not win]
-    - pkg-config  # [not win]
-    - cmake  # [win]
-    # latest build of 7.71.1 (build 6) is broken
-    - curl <7.71.1  # [win]
-    - m2-unzip  # [win]
+    - autoconf      # [not win]
+    - automake      # [not win]
+    - make          # [not win]
+    - libtool       # [not win]
+    - m4            # [not win]
+    - pkg-config    # [not win]
+    - cmake         # [win]
+    - curl          # [win]
+    - m2-unzip      # [win]
   host:
-    - giflib  # [not win]
+    # TODO: conda-forge giflib missing headers on windows?
+    - giflib        # [not win]
```

Comment:
An error on `win-64`:
```
  ERROR (leptonica): run-exports library package pkgs/main::jpeg-9e-h2bbff1b_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)

  ERROR (leptonica): run-exports library package pkgs/main::libwebp-1.2.0-h2bbff1b_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)

  ERROR (leptonica): run-exports library package pkgs/main::libpng-1.6.37-h2a8f88b_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)

  ERROR (leptonica): run-exports library package pkgs/main::libtiff-4.1.0-h8a3f274_2 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)

  ERROR (leptonica): run-exports library package pkgs/main::zlib-1.2.12-h8cc25b3_3 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```

It seems like `giflib`, `jpeg`, `libpng`, `libtiff`, `libwebp`, and `zlib` are not required on `win`:
```
    - giflib        # [not win]
    - jpeg          # [not win]
    - libpng        # [not win]
    - libtiff       # [not win]
    - libwebp       # [not win]
    - openjpeg
    - xz
    - zlib          # [not win]
```

see https://github.com/DanBloomberg/leptonica/blob/f4138265b390f1921b9891d6669674d3157887d8/CMakeLists.txt#L124-L133

Reply:
`Software Network (SW)` docs https://software-network.org/client/sw.pdf

Reply:
Hmm it uses this sw package manager. 
It would be better to build it with our own dependencies.
Have you tried building windows with -DSW_BUILD=OFF ?

Reply:
Thanks, I've disabled the SW package manager.